lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +74,47 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        file = json.load(open(self._path_to_config))
+
+        config = ConfigDTO(
+            seed_urls=file['seed_urls'],
+            headers=file['headers'],
+            total_articles_to_find_and_parse=file['total_articles_to_find_and_parse'],
+            encoding =file['encoding'],
+            timeout=file['timeout'],
+            should_verify_certificate=file['should_verify_certificate'],
+            headless_mode=file['headless_mode']
+        )
+        return config
+
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        config = json.load(open(self._path_to_config))
```

Comment:
use dto. otherwise, why do we need it

Reply:
How to use use it? In the documentation it is said:
> "This method should be called during Config class instance initialization step before Config._extract_config_content() method call to check config fields and make sure they are appropriate and can be used inside the program."
Actually, this way, I don't understand how to use dto, if we should to not to create it yet

Reply:
you should not put it in instance's attribute. nobody stops from making the instance inside the method and re-use only for this method.

Reply:
done